changelog/fix16997.dd

```diff
@@ -0,0 +1,19 @@
+-preview=intpromote switch is now set by default.
+
+integer promotions now follow C integral promotions rules consistently
+
+It affects operands of type byte, ubyte, short, ushort, char, and wchar. The operands are promoted to int before the operator is applied. The result type will now be int.
+
+The values computed will be different for some values of the operands:
+
+All values of - and ~ applied to ubyte, ushort, char, and dchar operands will change.
```

Comment:
This sentence doesn't read very well. "All results of ... applied to operands of type ubyte etc." perhaps

Reply:
done